st2-ask/cli-approve.txt

```diff
@@ -0,0 +1,33 @@
+vagrant@st2vagrant:~$ cat simple_approval.yaml
+---
+id: 59023cfc02ebd51154291652
+data:
+  approve: yes
+
+
+vagrant@st2vagrant:~$ st2 approval create simple_approval.yaml
++-------------+--------------------------------------------------------------+
+| Property    | Value                                                        |
++-------------+--------------------------------------------------------------+
+| id          | 59023cfc02ebd51154291652                                     |
+| data        | {                                                            |
+|             |     "approve": "yes"                                         |
+|             | }                                                            |
++-------------+--------------------------------------------------------------+
+
+vagrant@st2vagrant:~$ cat 2FA_approval.yaml
+---
+id: 59023cfc02ebd51154291653
+data:
+  second_factor: 012345
+
+
+vagrant@st2vagrant:~$ st2 approval create 2FA_approval.yaml
++-------------+--------------------------------------------------------------+
+| Property    | Value                                                        |
++-------------+--------------------------------------------------------------+
+| id          | 59023cfc02ebd51154291653                                     |
+| data        | {                                                            |
+|             |     "second_factor": "012345"                                |
+|             | }                                                            |
++-------------+--------------------------------------------------------------+
```

Comment:
Updated in https://github.com/StackStorm/st2tests/pull/104/commits/b0891dfb31f3a584aee81976ba3c58b22ff0d70a - I think the terminology of "response" is much better. Will strive to keep things consistent in this way going forward. "Approve" is just one type of response - response is the core primitive